llvm/lib/DebugInfo/LogicalView/Core/LVScope.cpp

```diff
@@ -2107,3 +2111,15 @@ bool LVScopeTemplatePack::equals(const LVScope *Scope) const {
 void LVScopeTemplatePack::printExtra(raw_ostream &OS, bool Full) const {
   OS << formattedKind(kind()) << " " << formattedName(getName()) << "\n";
 }
+
```

Comment:
Sorry, but we missed this one. It should be between `LVScopeFunctionType` and `LVScopeNamespace` to preserve the alphabetical order.

Reply:
Done :+1:.